modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -266,4 +273,29 @@ class MirrordApi(private val service: MirrordProjectService) {
         logger.error("mirrord stderr: $processStdError")
         throw Error("mirrord failed to start")
     }
+
+    /**
+     * Increments the mirrord run counter. Occasionally displays a notification asking for marketplace review.
+     */
+    private fun bumpFeedbackCounter() {
+        val previousRuns = MirrordSettingsState.instance.mirrordState.runsCounter
+        val currentRuns = previousRuns + 1
+
+        MirrordSettingsState.instance.mirrordState.runsCounter = currentRuns
+
+        if ((currentRuns % FEEDBACK_COUNTER_REVIEW_AFTER) != 0) {
+            return
+        }
+
+        service.notifier.notification(
+            "Enjoying mirrord? Don't forget to leave a review!",
```

Comment:
Same as in VSCode, lets also give the Feedback option here

Reply:
Done :+1: